libobs/obs-defs.h

```diff
@@ -41,6 +41,8 @@
 #define OBS_OUTPUT_UNSUPPORTED -6
 #define OBS_OUTPUT_NO_SPACE -7
 #define OBS_OUTPUT_ENCODE_ERROR -8
+#define OBS_OUTPUT_SLOBS_IDS -64
```

Comment:
Where/how is it used?

Reply:
Placeholder for error codes introduced for slobs. Same can be done by comment. But code more noticeable than comment.